tests/rest/admin/test_admin.py

```diff
@@ -643,3 +643,73 @@ def test_purge_room(self):
             self.assertEqual(count, 0, msg="Rows not purged in {}".format(table))
 
     test_purge_room.skip = "Disabled because it's currently broken"
+
+
+class UserCreateTestCase(unittest.HomeserverTestCase):
```

Comment:
this should be in a different file: `test_user.py`, to match `user.py` where the code inder test is being added.